packages/spanner/src/table.js

```diff
@@ -347,10 +349,10 @@ Table.prototype.insert = function(keyVals, callback) {
  *     be yielded. If using a composite key, provide an array within this array.
  *     See the example below.
  * @param {string=} query.index - The name of an index on the table.
+ * @param {number} query.limit - The number of rows to return.
```

Comment:
Ah, sorry for missing that. A quick thing we need to do is rename that parameter to match (`keyVals` => `query`). We can go with `query` as a name or `config`.

Beyond that, I think we should instead use `query.transactionOptions`, which future-proofs us when we add support for `query.gaxOptions`. Should we separate-issue that or do you want to get it done while we're in here?